src/index.html

```diff
@@ -13,6 +13,38 @@
     />
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day-sun calendar--month-length-31">
+      <div class="day calendar__day"></div>
```

Comment:
Remove it everywhere
```suggestion
      <div class="calendar__day"></div>
```